### ISSUE TITLE
Type helpers + @typescript-eslint/no-unused-vars

### ISSUE THREAD

**Issue by imagoiq** [drwpow/openapi-typescript#976]
**Description**

With the latest version (v6), the generated schema contains some type helpers even if there are not used which is catch as a unused variables.

```
12:5  error  'OneOf' is defined but never used. Allowed unused vars must match /_/u  @typescript-eslint/no-unused-vars
```

| Name                 | Version                      |
|:---------------------|:-----------------------------|
| `openapi-typescript` | `6.0.1`                      |
| Node.js              | `16.15.0`                      |
| OS + version         | `Windows 10` |

**Comment by drwpow** [comment]
I’d be open to a PR if someone would like to tackle this!
